WooCommerce/Classes/Extensions/OperatingSystemVersion+Localization.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+extension OperatingSystemVersion {
```

Comment:
Just wondering: Do we follow any rule when adding these extensions to `WooCommerce/Classes/Extensions` vs `WooFoundation` ?

Reply:
Not that I know of. My rule is that it goes in WooFoundation if it can – in this case it _can't_, because it has localised strings, which only get generated for the UI-level targets (WooCommerce, notifications, widgets)